imports/fixtures/octobersprintfixture.js

```diff
@@ -7,7 +7,7 @@ export const videos = [
     'price': 0,
```

Comment:
Can this file be renamed? Its current name seems oddly specific

Reply:
I don't understand ...

Reply:
I don't think you changed this file, but it originally showed up when this PR was targeting `master`. Just saying that the filename seemed a bit odd to me, but maybe it is entirely appropriate. Feel free to ignore my comment!